packages/js/src/Modules/Verto/webrtc/Peer.ts

```diff
@@ -16,7 +16,12 @@ import {
   RTCPeerConnection,
   streamIsValid,
 } from '../util/webrtc';
-import { isFunction } from '../util/helpers';
+import {
+  DeferredPromise,
```

Comment:
I think that `DeferredPromise` is not being used, just `deferredPromise` is

Reply:
Its just used as a type [here](https://github.com/team-telnyx/webrtc/pull/349/files#diff-266fd5c0ee3e801e3b5872c21adb485c9cd01703c6263ac297007cc24fd41f7eR33), but i noticed `debounce` in the next line was left unused after refactoring. Thanks for the catch